### PR TITLE
added support for managing the packagecloud repo's

### PR DESCRIPTION
**Pull request by mcrauwel**
added packagecloud module as a dependency

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+     ]
+    }
+  ],
+  "requirements": [
+    { "name": "json", "version_requirement": ">=1.8.0" },
+    { "name": "puppet", "version_requirement": ">= 3.2.0" }
+  ],
+  "dependencies": [
+    { "name": "puppetlabs/stdlib", "version_requirement": ">= 3.2.0 <5.0.0" },
+    { "name": "computology/packagecloud", "version_requirement": ">= 0.2.0" }
```
Comment (tomkrouper) [inline metadata.json]: Should this be [computology/computology-packagecloud](https://github.com/computology/computology-packagecloud)?

Comment (mcrauwel) [inline metadata.json]: i think it needs to be the reference to the puppet forge: https://forge.puppet.com/computology/packagecloud (which links back to the github repo you mentioned)

**Comment by tomkrouper** [comment]
@mcrauwel, can you add the assumption to the README.md with the new module required. Also, can you verify that you've tested. (We need to add automated testing on this at some point.) Once that is done, I think we are :+1:

**Comment by mcrauwel** [comment]
@tomkrouper I've added the new feature to the README.md with the note for the dependency.

I've also added a basic example file to verify the installation!

**Pull request merged**